package.json

```diff
@@ -29,7 +29,7 @@
     "test:build": "npm run build && node --experimental-modules --no-warnings -- ./node_modules/.bin/oletus test/build/*.js",
     "coverage:upload": "c8 report --reporter=text-lcov > coverage.lcov && codecov",
     "coverage:report": "c8 report --reporter=html",
-    "test:types": "tsc --lib es6 index.d.ts"
+    "test:types": "tsc index.d.ts"
```

Comment:
It's funny you're making this change now, seeing as it reverts a commit you made:

https://github.com/fluture-js/Fluture/commit/613291d41da3eac8e08c2f759d1680c35f509210

Reply:
Triple-slash directive already takes care of the `--lib` option

According to https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html

> For example, adding /// <reference lib="es2017.string" /> to one of the files in a compilation is equivalent to compiling with --lib es2017.string.

I checked out that commit which uses TS `v2.8.4`

It doesn't compile without `--lib`

```
> tsc index.d.ts

node_modules/@types/node/util.d.ts(115,88): error TS2304: Cannot find name 'Symbol'.
```

On that branch, some of the dependencies require `estree` which seems to be installing `@types/node` (this should be only installed in development)

`fluture.d.ts` doesn't need `--lib es6` option if I'm correct, if you don't pass that option then TypeScript somehow falls back to `node_modules/@types` (which shouldn't be there in the first place) and `util.d.ts` fails because es6 `Symbol` not defined.